litellm/llms/custom_httpx/http_handler.py

```diff
@@ -212,11 +219,18 @@ def __init__(
         if timeout is None:
             timeout = _DEFAULT_TIMEOUT
 
-        # Check if the HTTP_PROXY and HTTPS_PROXY environment variables are set and use them accordingly.
-        ssl_verify = bool(os.getenv("SSL_VERIFY", litellm.ssl_verify))
+        # SSL certificates (a.k.a CA bundle) used to verify the identity of requested hosts.
+        # /path/to/certificate.pem
+        ssl_verify = os.getenv(
```

Comment:
this change would revert a fix meant to handle SSL_VERIFY="True" being set as an env var @OgnjenFrancuski